src/relay/libp2p_transport_relay.erl

```diff
@@ -86,12 +86,13 @@ connect_to(_Pid, MAddr, Options, Timeout, TID) ->
             Error;
         {ok, SessionPid} ->
             Swarm = libp2p_swarm:swarm(TID),
-            {ok, _} = libp2p_relay:dial_framed_stream(
+            {ok, Stream} = libp2p_relay:dial_framed_stream(
                 Swarm,
                 RAddress,
                 [{type, {bridge_cr, MAddr}}]
```

Comment:
can you pass self() here and avoid having to lookup the pid to message back?

Reply:
I think that you can do it without passing.  This goes through several other `dial_framed_stream` calls, but the last one could call self() before spawning.

Reply:
I think this is the call at the bottom and we're still in the dialing process: https://github.com/helium/erlang-libp2p/blob/master/src/libp2p_session.erl#L75

I just suggest this because it might be an easier change than threading self through all the intervening layers.

Reply:
You can also just prefix/suffix it to the parameter list there and it will get passed through...

Reply:
I looked into doing this, and it seems quite a bit more complicated. Propose we skip it for now.